src/test/skript/tests/syntaxes/effects/EffReplace.sk

```diff
@@ -43,7 +43,7 @@ test "replace strings":
 	assert {_text::8} is "My name is bob and this is my friend bob!" with "replace first pattern 1 (case sensitivity enabled) (%{_text::7}%) failed"
 
 	# Replacing in an ExpressionList
-	set {_list::*} to "1", "2", "12"
+	set {_list::*} to "1", "2", and "12"
```

Comment:
```suggestion
	set {_list::*} to "1", "2" and "12"
```

Reply:
`, and` is actually proper English and preferred, but only Njol examples in Skript follow.